code/items/equipments/entities/HealthstationEntity.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Items
+{
+    using Sandbox.UI.Construct;
+
+    using UI;
```

Comment:
move it up

Reply:
yeah I need to fix this within our editor config, will do that. Thanks for the catch.